cli/jobs/single-step/r/accidents/src/aml.R

```diff
@@ -0,0 +1,110 @@
+library(mlflow)
```

Comment:
have we resolved the issue with the token refresh not happening while the R process is running? The scenario was to run a very long job by means of 
```R
for(i in 1:60) {
    Sys.sleep(60)
}
```

Reply:
@danielsc  Yes that issue is resolved. We are using tcktk2 R library (which depends on [tcl tk](https://www.tcl.tk/) ) to periodically refresh token.